provider/aws/formation/app.json.tmpl

```diff
@@ -248,7 +275,10 @@
   {{ range .Manifest.Services }}
     "Service{{ upper .Name }}Service": {
       "Type": "AWS::ECS::Service",
-      {{ if .Port.Port }}
```

Comment:
This chunk seems incorrect. It would only create the service if DefaultDomain=Yes?

I don't think this branch is necessary.

Reply:
From my understanding, if user specifies `Port` in convox.yml, and set the `DefaultDomain` to `No`, then the ECS Service will have dependency problem due to no ALB listener rule created?

Reply:
With this code if the user specifies `port:` but not `domain:` and sets `DefaultDomain=No` the service would not be created.

Reply:
Yep, since in that case, no ALB listener rules will be created, so (correct me if I'm wrong) it's impossible to add Load Balancer to ECS service. 

Do we want to create the ECS service without Load Balancer? Any better idea?